rfcs/0002-machine-state-catchup.rst

```diff
@@ -0,0 +1,173 @@
+Fast, incremental radicle app state
+===================================
+
+.. date:: 2018-12-19
+.. reviewers::
+
+   @geigerzaehler
+   @MeBrei
+   @jkarni
+
+Synopsis
+---------
+
+This RFC proposes full serialisation of the radicle interpreter state as a
+solution to maintaining up-to-date views/projections of the state of a
+long-running machine that is relevant to an app. For facilitating deriving these
+projections, it proposes that machine outputs are write-commands for a
+particular database.
+
+Motivation
+----------
+
+An *app* (e.g. the radicle issues CLI) is a program which reads data from a
+radicle machine, displays it to the user, and facilitates ``send!``ing inputs to
```

Comment:
RST does not support this :(. You need to leave a space between the closing <code>``</code> and “ing”.

Reply:
:sob: